src/components/HealthCardDisplay/HealthCardDisplay.js

```diff
@@ -206,39 +229,78 @@ const HealthCardDisplay = () => {
-                <Typography variant="h6" className={issuerVerified ? styles.verifiedText : styles.unverifiedText}>
-                  {issuerVerified ? t('healthcarddisplay.Issuer recognized') : t('healthcarddisplay.Issuer not recognized')}
+                <Typography
+                  variant="h6"
+                  className={
+                    issuerVerified.every((e) => e === true)
+                      ? styles.verifiedText
+                      : styles.unverifiedText
+                  }
+                >
+                  {issuerVerified.every((e) => e === true)
+                    ? t('healthcarddisplay.Issuer recognized')
+                    : t('healthcarddisplay.Issuer not recognized')}
                 </Typography>
               </Box>
             </Box>
           </Grid>
         )}
       </Grid>
-      {(!patientData || !bottomBannerStyle) ? (
+      {!patientDataArray || !bottomBannerStyle ? (
         <Grid container className={styles.errorRoot}>
           <Grid item md={6} display="flex" justifyContent="center">
-            <img src={errorIllustration} alt="Error Illustration" width="100%" />
+            <img
+              src={errorIllustration}
+              alt="Error Illustration"
+              width="100%"
+            />
           </Grid>
-          <Grid item container md={6} className={styles.errorGrid} rowSpacing={{ xs: '1rem', md: '2rem' }}>
+          <Grid
+            item
+            container
+            md={6}
+            className={styles.errorGrid}
+            rowSpacing={{ xs: '1rem', md: '2rem' }}
+          >
             <Grid item>
               <Typography textAlign="center" variant="h4" color="primary.main">
                 <Trans
                   i18nKey="healthcarddisplay.Only valid SMART Health Card QR Codes are currently supported."
                   components={[
-                    <span className={styles.shcText}> SMART&reg; Health Card </span>
+                    <span className={styles.shcText}>
+                      {' '}
+                      SMART&reg; Health Card
+                      {' '}
+                    </span>,
                   ]}
                 />
               </Typography>
             </Grid>
             <Grid item>
-              <QrScanButton onClick={handleScan} styles={{ width: '100%' }} mt={10} />
+              <QrScanButton
+                onClick={handleScan}
+                styles={{ width: '100%' }}
+                mt={10}
+              />
             </Grid>
           </Grid>
         </Grid>
       ) : (
         <Grid item className={styles.flexCard}>
+          <Box>
+            <Router>
+              <Switch>
+                <Link to="/qr-scan" onClick={handleScanWithMultipleHC}>
+                  {t('healthcarddisplay.Multiple SMART Health Cards')}
+                </Link>
+              </Switch>
+            </Router>
+          </Box>
           <VaccineCard padding="1rem" width="100%" />
```

Comment:
There doesn't seem to be padding between the different cards, would add a little space.